packages/playout-gateway/src/tsrHandlerJobs/removeDeviceJob.ts

```diff
@@ -0,0 +1,33 @@
+import { CoreTSRDeviceHandler } from '../coreHandler'
+import { Logger } from 'winston'
+import { Job } from './job'
+
+export class RemoveDeviceJob extends Job<void> {
+	protected artifacts: undefined
+
+	constructor(
+		private deviceId: string,
+		private coreTsrHandlers: { [deviceId: string]: CoreTSRDeviceHandler },
+		private expected: boolean,
+		private logger: Logger
+	) {
+		super()
+	}
+
+	async run(): Promise<void> {
+		if (this.coreTsrHandlers[this.deviceId]) {
+			try {
+				if (this.expected) {
```

Comment:
Could be a ternary.

``` typescript
const dispose = this.expected
  ? this.coreTsrHandlers[this.deviceId].disposeExpectedly
  : this.coreTsrHandlers[this.deviceId].disposeUnexpectedly
await dispose()
```
Should shrink the amount of lines used by one or two lines. 
Could move the call to the same line as the ternary, with use of parenthesis, if wanted.

``` typescript
await (this.expected
  ? this.coreTsrHandlers[this.deviceId].disposeExpectedly
  : this.coreTsrHandlers[this.deviceId].disposeUnexpectedly)()
```

Reply:
Does it benefit from being less lines? 
I think that makes it a lot harder to read

Especially the second version, where if skim reading, you can easily miss that disposeExpectedly is a function